.github/workflows/build.yml

```diff
@@ -10,25 +10,22 @@ on:
 
 jobs:
   Build:
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
Does this have to be windows?

Reply:
Yes, because Desktop MSBuild is required for the Android and iOS feature sets.